src/material-experimental/mdc-list/list-base.ts

```diff
@@ -6,6 +6,34 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
+import {AfterContentInit, ElementRef, NgZone, OnDestroy, QueryList} from '@angular/core';
+import {setLines} from '@angular/material/core';
+import {Subscription} from 'rxjs';
+import {startWith} from 'rxjs/operators';
+
 export class MatListBase {}
 
-export class MatListItemBase {}
+export class MatListItemBase implements AfterContentInit, OnDestroy {
+  lines: QueryList<ElementRef<Element>>;
+
+  private _subscriptions = new Subscription();
+
+  constructor(protected _element: ElementRef, protected _ngZone: NgZone) {}
+
+  ngAfterContentInit() {
+    this._ngZone.runOutsideAngular(() => {
```

Comment:
Can you move this into its own function that's called from `ngAfterContentInit`? That would give it a name that described what's it's doing